src/controllers/csi/metadata/sqlite.go

```diff
@@ -177,6 +184,19 @@ func (a *SqliteAccess) createTables() error {
 	return nil
 }
 
+// setupDynakubeTable creates the dynakubes table if it doesn't exist and tries to add additional columns
+func (a *SqliteAccess) setupDynakubeTable() error {
+	if _, err := a.conn.Exec(dynakubesCreateStatement); err != nil {
+		return fmt.Errorf("couldn't create the table %s, err: %s", dynakubesTableName, err)
+	}
+
+	if _, err := a.conn.Exec(dynakubesAlterStatementUsesImageColumn); err != nil {
+		// statement errors if column already exists, swallow error
+		log.Info("column UsesImage already exists")
```

Comment:
what if there is an error for a different reason?

Reply:
seems I missed that the [error code](http://www.sqlite.org/c3ref/c_abort.html) and [extended error code](http://www.sqlite.org/c3ref/c_abort_rollback.html) actually indicate a `SQLITE_ERROR` with `SQLITE_ERROR_MISSING_COLLSEQ`:
![image](https://user-images.githubusercontent.com/33160506/172811800-b4a4372a-9844-4831-9d20-37cbae8af600.png)

[error_missing_collseq](https://www.sqlite.org/rescode.html#error_missing_collseq) seems to be an issue specific to alter/create, so much safer to only skip in that case.